magenta/music/notebook_utils.py

```diff
@@ -132,11 +132,15 @@ def _sequence_to_pandas_dataframe(sequence):
 def download_bundle(bundle_name, target_dir, force_reload=False):
   """Downloads a Magenta bundle to target directory.
 
+  Target directory target_dir will be created if it does not already exist.
+
   Args:
      bundle_name: A string Magenta bundle name to download.
      target_dir: A string local directory in which to write the bundle.
      force_reload: A boolean that when True, reloads the bundle even if present.
   """
+  if not os.path.exists(target_dir):
+    os.makedirs(target_dir)
```

Comment:
Should just call:
```
tf.gfile.MakeDirs(target_dir)
```

The gfile version has better internal compatibility and will just return if the directory already exists, so no need to do the os.path.exists check.